recipes/taskipy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "taskipy" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0006429f708f530fc7add28ca51fd41f6c6e1fbe86763ff05125e1af3e8bdc7e
+
+build:
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - task = taskipy.cli:main
+
+requirements:
```

Comment:
is there a way to sync the requirements to the main package's directly? I'm asking since reqs may be added \ updated \ removed on a patch-minor version basis

Reply:
This step is semi-automated: When a new version is released on PyPI, conda-forge bots will open a PR with the relevant package version/hash changes and verify if any new dependencies have been added. It's up to the maintainer to ensure that the versions of existing dependencies are in sync with upstream.